pmg/templates/admin/my_edit.html

```diff
@@ -31,10 +30,10 @@
 		</div>
 	    </div>
 	</div>
-	<script defer>
-	 $(document).ready(function(){
+	<script>
+	 window.onload = function(){
```

Comment:
Use `window.addEventListener("load",...` instead, otherwise this might replace another event handler and we'll scratch out heads for ever.

https://stackoverflow.com/a/16683250/1305080

Reply:
Can you further explain. Its not clear on SO